R/create-package-docs.R

```diff
@@ -0,0 +1,165 @@
+#' @title Create validation docs
+#'
+#' @param product_name The product being validated.
+#' @param version The version number of the product.
+#' @param repo_url Character string denoting the url of repository.
+#' @param specs tibble of stories and requirements. See [input_formats].
+#' @param release_notes_file file path to a formatted markdown doc of release notes.
+#' @param style_dir Directory that has style references for the generated docx
+#'   files. When generating each output file, Pandoc will be instructed to use
+#'   the reference file from this directory that has the same base name (if it
+#'   exists).
+#' @param output_dir Directory to write the output documents to. Defaults to
+#'   working directory.
+#' @param write Whether to create the output docs. Setting this to `FALSE` is
+#'   useful when you're just interested in the return value.
+#' @param cleanup_rmd Whether to delete the copied RMD's after the word documents are generated.
+#'  Defaults to `TRUE`.
```

Comment:
I removed the `auto_test_dir` arg above:
```r
#' @param auto_test_dir path to directories containing automatic
#'   test output files. See [input_formats].
```

I was getting an error about there being duplicate `auto_test_dir` entries during the devtools check, and this seemed to make it go away. The docs and functions still look and operate fine, but I find it odd it shouldnt be defined here. Perhaps I could just add `@param auto_test_dir`, but its Friday and I have a feeling that wouldnt work

Reply:
The problem is that you have `#' @describeIn create_package_docs` in the `create_metworx_docs()` entry above. That combines these into a single doc, which creates the duplicate param entry. 

I'm fixing this (and a few other little things) in #84.

Reply:
@seth127 yeah I wanted to include them in the same doc because they're so similar. If they had the same definition it would probably have been fine, but the wording changed a bit between the two functions. Im fine to separate them into separate docs though

Reply:
I think we should separate them (I'll do it in #84) mostly because, even if the implementation is similar, the users (who are the ones consuming the docs) will be different.